_ClientSideUI/index.html

```diff
@@ -336,14 +336,14 @@
     var metricsDef;        //holds the definition of the metrics (from server side metrics.js)
     var eventsDef;
     var showHiddenNodes=false, showRawSend=false;
-    var socket = io.connect('<?php echo $_SERVER['HTTP_HOST']; ?>', {'connect timeout': 1000}); //limit chrome xhr-polling fall back delay
```

Comment:
I just compared lines visually and assumed that was some kind of unintentional edit, but you are correct, the simple "var socket = io()" will work. Also loading socket.io js files from local should be a pretty good idea.